hat/assets/js/apps/Iaso/domains/instances/components/InstancesFiltersComponent.js

```diff
@@ -563,7 +564,11 @@ const InstancesFiltersComponent = ({
                             </Box>
                         )}
                         <Button
-                            disabled={textSearchError || isSearchDisabled}
+                            disabled={
+                                textSearchError ||
+                                (params?.isSearchActive === 'true' &&
+                                    isSearchDisabled)
```

Comment:
SearchActive is true and Search is Disabled at the same time. The naming is a bit confusing here. If I understand well, the check on SearchActive is to "counter" the check on `!isInstancesFilterUpdated`, but the result is a bit confusing to read 😅